config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
 
 # Target score for pipeline.py:
 0
```

Comment:
you have to specify expectations so that pipeline tests could start and check your implementation